net-libs/opendht/opendht-9999.ebuild

```diff
@@ -24,14 +24,19 @@ LICENSE="GPL-3"
 
 SLOT="0"
 
-IUSE="python static-libs tools"
+IUSE="doc python static-libs tools"
 
-DEPEND=">=dev-libs/msgpack-2.0
+DEPEND=">=app-crypt/argon2-20161029
+	>=dev-libs/msgpack-2.0
```

Comment:
in opendht now is bundled argon2-2016.10.25 https://github.com/savoirfairelinux/opendht/commit/28a5740c3754f6916416e7399c4b27b51a97928e